docs/ruleset_functions/get_lighting_status_type.md

```diff
@@ -0,0 +1,31 @@
+
+## get_lighting_status_type
+
+Description: This function would determine whether the space lighting status type is not-yet designed, as-designed or as-existing.  
+
+Inputs:
+  - **building_segment**: The building segment that needs to determine lighting status type.  
+
+Returns:
+- **space_lighting_status_type**: The Lighting Status Type Category [not-yet designed, as-designed or as-existing].    
+
+Data Lookup:
+- Table 9.5.1  
+
```

Comment:
We should add a clarification somewhere that this function ONLY applies to the proposed building.

Reply:
Agreed. One hole in our logic here is when lighting is existing or fully designed, but the LPD by chance is exactly the same as BAM LPD. Perhaps capture this scenario in the error message that the calling RDS would show.

Reply:
Added this to the returned values.